xtreg2way.m

```diff
@@ -67,7 +67,7 @@
 switch se
     case 0
         sig2hat=(reg.res'*reg.res)/(sum(struc.w>0)-struc.N-struc.T+1-numel(reg.beta));
-        aVarHat=sig2hat/reg.XX;
+        aVarHat=sig2hat*inv(reg.XX);
```

Comment:
Good change! It was crashing before, right?

Reply:
Yes. But now it´s working perfect!